consensus/ibft/ibft.go

```diff
@@ -965,15 +969,14 @@ func (i *Ibft) runValidateState() {
 // updateMetrics will update various metrics based on the given block
 // currently we capture No.of Txs and block interval metrics using this function
 func (i *Ibft) updateMetrics(block *types.Block) {
+	// get previous header
 	prvHeader, _ := i.blockchain.GetHeaderByNumber(block.Number() - 1)
-	parentTime := time.Unix(int64(prvHeader.Timestamp), 0)
-	headerTime := time.Unix(int64(block.Header.Timestamp), 0)
-	//Update the block interval metric
-	if block.Number() > 1 {
-		i.metrics.BlockInterval.Observe(
-			headerTime.Sub(parentTime).Seconds(),
-		)
-	}
+	// calculate difference between previous and current header timestamps
+	diff := time.UnixMilli(int64(block.Header.Timestamp)).Sub(time.UnixMilli(int64(prvHeader.Timestamp)))
```

Comment:
same as well

Reply:
Resolved backward compatibility with two custom functions in consensus/utils.go that do the same thing 631515404cf370d1fc015938ea35da52ff9dbaaf

Reply:
You're linking the wrong commit but the change is there :) @ZeljkoBenovic  Resolving comment